src/mlpack/methods/ann/layer/batch_norm_impl.hpp

```diff
@@ -216,26 +216,26 @@ void BatchNormType<MatType>::Forward(
     // Calculate mean and variance over all channels.
     MatType mean = arma::sum(arma::sum(inputTemp, 2), 0) / m;
     variance = arma::sum(arma::sum(arma::pow(
-        inputTemp.each_slice() - arma::repmat(mean,
+        inputTemp.each_slice() - repmat(mean,
```

Comment:
You can probably bring `repmat(mean, inputSize, 1)` all on one line now---probably easier to read too.